cmd/integrationArtifactUpload.go

```diff
@@ -193,11 +193,12 @@ func GetJSONPayloadAsByteArray(config *integrationArtifactUploadOptions, mode st
 	}
 	jsonObj := gabs.New()
 	if mode == "create" {
-		jsonObj.Set(config.IntegrationFlowID, "Name")
+		jsonObj.Set(config.IntegrationFlowName, "Name")
```

Comment:
In the past the IntegrationFlowName was not used throughout the step (even though it is a mandatory parameter). Now it is necessary? Because the create URL and method have not changed, why does the payload have to?

Reply:
there was bug earlier i.e. config.IntegrationFlowID was used for both name and id, which is wrong and i changed that.
you can refer here for more details on API usage :-

 https://api.sap.com/api/IntegrationContent/resource 
API - (Create/upload an integration flow.)
/IntegrationDesigntimeArtifacts